tools/CLU/BuildDrop.ps1

```diff
@@ -9,9 +9,6 @@ if (!($workspaceDirectory))
     $env:WORKSPACE = $workspaceDirectory
 }
 
-$buildProfileScriptPath = "`"$thisScriptDirectory\BuildProfile.ps1`"" # Guard against spaces in the path
-$sourcesRoot = "$workspaceDirectory\src\clu"
```

Comment:
i move this after the folder creating script lines, because buildprofile.ps1 appears want to ensure the folder existence.